LiftLog.Lib/Models/SessionModels.cs

```diff
@@ -162,6 +163,6 @@ bool PerSetWeight
     public decimal MaxWeight => PotentialSets.Select(x => x.Weight).DefaultIfEmpty(0).Max();
 }
 
-public record RecordedSet(int RepsCompleted, TimeOnly CompletionTime);
+public record RecordedSet(int RepsCompleted, DateOnly CompletionDate, TimeOnly CompletionTime);
```

Comment:
```suggestion
public record RecordedSet(int RepsCompleted, DateOnly CompletionDate, TimeOnly CompletionTime)
{
    public readonly DateTime CompletionDateTime => CompletionDate.ToDateTime(CompletionTime, DateTimeKind.Local);
}
```

This seems to be done in a few places, might be good to just compute it in a single place